src/Microsoft.DotNet.Interactive/JavascriptUtilities.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.DotNet.Interactive.Http
+{
+    public static class JavascriptUtilities
+    {
+        public static string EnsureRequireJs(Uri requireJsUri = null, string callBackName = null)
```

Comment:
This name and return type could be clearer. `Ensure*` usually means ensure some thing has been done, and signals idempotency. This returns a `string`.

What's it really doing?

Reply:
that returns the javascript code snippet that checks and pull in if needed the requirejs library